package.json

```diff
@@ -61,30 +61,22 @@
     "supply": "0.0.4",
     "urijs": "^1.18.7"
   },
-  "peerDependencies": {
+  "optionalDependencies": {
```

Comment:
why ?

Reply:
`peerDependencies` must be installed in connectors packages even if the connector does not use it